docs/collecting-data/collecting-from-own-applications/react-native-tracker/tracking-events/index.md

```diff
@@ -197,11 +196,21 @@ tracker.trackScreenViewEvent({
 
 - `id`: (string) - The id(UUID) of screen that was viewed
 - `type`: (string) - The type of screen that was viewed
-- `previousName`: (string) - The name of the previous screen that was viewed
-- `previousId`: (string) - The id(UUID) of the previous screen that was viewed
-- `previousType`: (string) - The type of screen that was viewed
+- `previousName`: (string) - The name of the previous screen that was viewed (assigned automatically)
+- `previousId`: (string) - The id(UUID) of the previous screen that was viewed (assigned automatically)
+- `previousType`: (string) - The type of screen that was viewed (assigned automatically)
 - `transitionType`: (string) - The type of transition that led to the screen being viewed
 
+#### Tracking screen views in React Navigation
```

Comment:
Since these sections refer to same tracker method, would it make sense to have a level 5 heading here to nest these sections under `#### Screen View` instead of same level?